test/integration/lib/config/foxbox.js

```diff
@@ -0,0 +1,28 @@
+module.exports = {
+
+	nupnp_server : {
```

Comment:
Nit: The rest of the project uses spaces instead of tab. I'd make the PR comply to it.